common/test/acceptance/tests/lms/test_lms_user_preview.py

```diff
@@ -266,28 +266,26 @@ def populate_course_fixture(self, course_fixture):
             </problem>
         """)
 
+        self.alpha_text = "VISIBLE TO ALPHA"
```

Comment:
Note to reviewers: making all caps to make the string comparison easier. Currently the LMS uppercases display names. By making the names uppercase initially, we don't have to worry about that (and if the LMS stops uppercasing display names, the test won't need to change).